src/vsr/replica.zig

```diff
@@ -309,7 +309,6 @@ pub fn ReplicaType(
         /// * never decreases.
         /// Invariants (status=normal primary):
         /// * `replica.commit_max = replica.commit_min`.
-        /// * `replica.commit_max = replica.op - pipeline.queue.prepare_queue.count`.
```

Comment:
Where does this not hold?

Reply:
Ah, forgot to mention that I was confused by this. This almost never holds? For example, in an idle cluster `replica.commit_max = replica.op`. Or am I misreading this?

Reply:
`pipeline.queue.prepare_queue.count` is the number of items currently in the pipeline, not the capacity. So in an idle cluster `pipeline.queue.prepare_queue.count == 0`.

Reply:
Ok, now advance_commit_max actually checks that invariant (had to move pipeline prepare popping to commit_op for that)